frontend/src/ts/commandline/index.ts

```diff
@@ -382,21 +382,11 @@ function restoreOldCommandLine(sshow = true): void {
   if (sshow) show();
 }
 
-$("#commandLine input").keyup((e) => {
+$("#commandLine input").on("input", (e) => {
```

Comment:
My bad again. Please remove the `e` that I added as a parameter. Everything else seems fine.

```suggestion
$("#commandLine input").on("input", () => {
```